app/services/permission_service.rb

```diff
@@ -31,6 +31,7 @@ def platform_admin_permissions(controller, action)
       return true if controller == 'admin/locations' && action.in?(%w(index show))
       return true if controller == 'users'           && action.in?(%w(show edit update profile))
       return true if controller == 'stores'          && action.in?(%w(index edit validate show))
+      return true if controller == 'orders'          && action == 'show'
```

Comment:
Operator `&&` should be surrounded with a single space.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.